docs/devices/9290030675.md

```diff
@@ -86,7 +86,7 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 Value can be found in the published state on the `motion_sensitivity` property.
 To read (`/get`) the value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"motion_sensitivity": ""}`.
 To write (`/set`) a value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"motion_sensitivity": NEW_VALUE}`.
-The possible values are: `low`, `medium`, `high`.
+The possible values are: `very low`, `low`, `medium`, `high`, `very high`.
```

Comment:
very low/high is not allowed: https://github.com/Koenkk/zigbee-herdsman-converters/blob/6ee7e4fcff30ea45b7be43bbf50cb5249b4ca5c1/converters/toZigbee.js#L2112

Reply:
Forgot this comment, just noticed https://github.com/Koenkk/zigbee-herdsman-converters/pull/4759. This part is automatically updated if exposes is updated so this pr is not needed.